object-common-multipart.go

```diff
@@ -458,11 +458,11 @@ func listMultipartUploadsCommon(layer ObjectLayer, bucket, prefix, keyMarker, up
 		if fi.Mode.IsDir() {
 			// All directory entries are common prefixes.
 			uploadID = "" // Upload ids are empty for CommonPrefixes.
-			objectName = strings.TrimPrefix(fi.Name, retainSlash(path.Join(mpartMetaPrefix, bucket)))
+			objectName = strings.TrimPrefix(fi.Name, pathJoin(mpartMetaPrefix, bucket))
```

Comment:
this should be `retainSlash()` as well because we want to trim `multipart/bucket/` i.e with trailing `/`